src/main/java/aspects/MetricsAspect.java

```diff
@@ -115,7 +115,7 @@ private long getRestoreBlockedTime() {
         if (restoreFactory.getDownloadRestoreTimer() == null) {
             return 0;
         }
-        return restoreFactory.getDownloadRestoreTimer().durationInMs();
```

Comment:
this method gets used in the calculation above for `getBlockedTime` which seems to be expecting this value in ms.

Reply:
wrong flag. resolving.